build-tools/packages/build-cli/src/commands/check/policy.ts

```diff
@@ -193,7 +193,7 @@ export class CheckPolicy extends BaseCommand<typeof CheckPolicy> {
 				filePathsToCheck.push(...stdInput.split("\n"));
 			}
 		} else {
-			const repo = new Repository({ baseDir: gitRoot });
+			const repo = new Repository({ baseDir: gitRoot }, "microsoft/FluidFramework");
```

Comment:
What does this mean for devs' forks? Having this hard-coded seems odd at first glance.

Reply:
+1

Reply:
This is used to locate the upstream remote, which can have an arbitrary name. Code that expects to find an upstream
matching this will fail when run on a repo without a matching remote.

Reply:
This is a somewhat common pattern in the release-related tools especially. I'm not too concerned about this since code
already had to deal with a missing remote.